mix.exs

```diff
@@ -60,6 +60,7 @@ defmodule FunWithFlags.Mixfile do
     [
       {:redix, "~> 1.0", optional: true},
       {:ecto_sql, "~> 3.0", optional: true},
+      {:ecto_sqlite3, "~> 0.8.2", optional: true, only: [:dev, :test]},
```

Comment:
I think that just the minor version should be fine. It would be consistent with the others.
```suggestion
      {:ecto_sqlite3, "~> 0.8", optional: true, only: [:dev, :test]},
```